crates/astria-sequencer/src/genesis.rs

```diff
@@ -18,9 +24,54 @@ pub(crate) struct GenesisState {
 }
 
 impl GenesisState {
-    pub(crate) fn from_path<P: AsRef<Path>>(path: P) -> anyhow::Result<Self> {
-        let file = File::open(path).context("failed to open file with genesis state")?;
-        serde_json::from_reader(&file).context("failed deserializing genesis state from file")
+    /// Load Account information from a CSV file
+    pub(crate) fn from_csv<P: AsRef<Path>>(path: P) -> anyhow::Result<Self> {
+        let mut reader = Reader::from_path(path)?;
+        let mut records: Vec<Account> = Vec::new();
+
+        for result in reader.deserialize() {
+            let record: Account = result?;
+            records.push(record);
+        }
+
+        Ok(Self {
+            accounts: records,
+        })
+    }
+
+    /// Add the Account data from `GenesisState` to the `CometBFT` genesis.json file
+    pub(crate) fn propagate_accounts_to(&self, path: &str) -> anyhow::Result<()> {
```

Comment:
So this overrides the cometbft genesis file from under it? I don't know how I feel about this. IMO `sequencer` third tools should never change files in the config/cache/etc directories of other tools.

Reply:
pretty sure this is what the cosmos-sdk does :/ maybe we should copy the cometbft default genesis file and write to that?